src/components/FAQ.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Collapse, Typography } from 'antd';
+import '../styles/FAQ.less';
+import { FAQ_data } from '../content/FAQContent';
+import { MinusOutlined, PlusOutlined } from '@ant-design/icons';
+
+const FAQ: React.FC = () => {
+  return (
+    <div className="faq-container">
+      <Collapse
+        expandIcon={({ isActive }) => (isActive ? <MinusOutlined /> : <PlusOutlined />)}
+        expandIconPosition="right"
+      >
+        {FAQ_data.map((f, i) => (
+          <Collapse.Panel key={i} header={<Typography.Text strong>{f.title}</Typography.Text>}>
```

Comment:
Title text size is wrong, can you try one of the other "levels" of Typography as provided by antd to get it closer to the Zeplin designs.

Reply:
There is no levels for `Typography.Text` so instead I will directly change the `font-size` in less